sql/updates/world/3.3.5/2020_99_99_00_world.sql

```diff
@@ -0,0 +1,36 @@
+-- Creature Sneed 643 SAI - Add Set instance data line
+SET @ENTRY := 643;
+UPDATE `creature_template` SET `AIName`="SmartAI" WHERE `entry`= @ENTRY;
+DELETE FROM `smart_scripts` WHERE `entryorguid`=@ENTRY AND `source_type`=0;
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(@ENTRY, 0, 0, 0, 0, 0, 100, 2, 7900, 17600, 62000, 86200, 11, 6713, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, "Every 62 - 86.2 secs (7.9 - 17.6s initially) [IC] - Self: Cast spell 6713 on Victim // Sneed - In Combat - Cast 'Disarm' (No Repeat) (Normal Dungeon)"),
+(@ENTRY, 0, 1, 0, 6, 0, 100, 0, 0, 0, 0, 0, 34, 2, 3, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, "On death - Set instance data #2 to 3 // Sneed - On Just Died - Set Instance Data #2 to 3 (DONE)");
+
+-- Creature Gilnid 1763 SAI - Add Set instance data line
+SET @ENTRY := 1763;
+UPDATE `creature_template` SET `AIName`="SmartAI" WHERE `entry`= @ENTRY;
+DELETE FROM `smart_scripts` WHERE `entryorguid`=@ENTRY AND `source_type`=0;
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(@ENTRY, 0, 0, 0, 1, 0, 100, 2, 120000, 120000, 120000, 120000, 1, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, "When out of combat and timer at the begining between 120000 and 120000 ms (and later repeats every 120000 and 120000 ms) - Self: Talk 0 // Gilnid - Out of Combat - Say Line 0 (No Repeat) (Normal Dungeon)"),
+(@ENTRY, 0, 1, 0, 0, 0, 100, 2, 2100, 4800, 23300, 44900, 11, 5213, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, "Every 23.3 - 44.9 secs (2.1 - 4.8s initially) [IC] - Self: Cast spell 5213 on Victim // Gilnid - In Combat - Cast 'Molten Metal' (No Repeat) (Normal Dungeon)"),
+(@ENTRY, 0, 2, 0, 6, 0, 100, 0, 0, 0, 0, 0, 34, 3, 3, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, "On death - Set instance data #3 to 3 // On just died - Set instance data #3 to 3 (DONE)");
```

Comment:
If you will keep this code, please change it to:
```sql
DELETE FROM `smart_scripts` WHERE `entryorguid` = 643 AND `source_type` = 0 AND `id` = 1;
DELETE FROM `smart_scripts` WHERE `entryorguid` = 1763 AND `source_type` = 0 AND `id` = 2;
INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
(643, 0, 1, 0, 6, 0, 100, 0, 0, 0, 0, 0, 34, 2, 3, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, "Sneed - On Death - Set Instance Data 2 to 3 (DONE)"),
(1763, 0, 2, 0, 6, 0, 100, 0, 0, 0, 0, 0, 34, 3, 3, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, "Gilnid - On Death - Set Instance Data 3 to 3 (DONE)");
```

Reply:
Thanks for your review and feedback on the SQL changes. I will first try to change the script according to the example given by @Treeston.